handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -287,6 +289,18 @@ protected void deallocate() {
         keyMaterialManager = context.keyMaterialManager();
     }
 
+    /**
+     * Enables (or disables) OCSP stapling on the {@link SSLEngine}.
+     */
+    @UnstableApi
+    public synchronized boolean enableOcsp(boolean enabled) {
+        if (ssl != 0L) {
```

Comment:
`SSL.setStatusType` protects against null `SSL*` too ... can we just rely on the null there instead of doing it in both places?